.github/stale.yml

```diff
@@ -0,0 +1,15 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 3
```

Comment:
3 days seems a bit short for marking things as stale, I would suggest perhaps 5 days ?